src/main/java/ch/sbb/SbbChHomePage.java

```diff
@@ -0,0 +1,51 @@
+package ch.sbb;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class SbbChHomePage {
+    WebDriver driver;
```

Comment:
no access modifiers means that all your variables are package-private. it's worth considering an access strategy and think through if this option is what you really want.
in a project like this, most probably you will not face any issues with it, but we can still be a bit more restrictive.
since your page objects encapsulate the locators, I don't see any reasons why not keep them private usually.

Reply:
you are right, I'll try to make a habit checking the access modifiers
fixed in this particular case